src/classes/Metrics.js

```diff
@@ -370,7 +370,9 @@ class Metrics {
 
 			// New parameter for Ghostery 8.5.2
 			// Hub Promo variant
-			`&hp=${encodeURIComponent(Metrics._getHubPromoVariant().toString())}`;
+			`&hp=${encodeURIComponent(Metrics._getHubPromoVariant().toString())}` +
+			// Subscription Interval
+			`&si=${encodeURIComponent(Metrics._getSubscriptionInterval().toString())}`;
```

Comment:
There's a matching param on the `metrics-server` side or a PR to add one, yah?

Reply:
Yup, the PR is here. https://github.com/ghostery/metrics_server/pull/25. Good catch, I think we need to update the param name to `subscription_interval`. Let's do that here to get 8.5.2 out the door quicker

Reply:
Good idea